tests/run-make/rustdoc-error-lines/rmake.rs

```diff
@@ -4,9 +4,8 @@
 use run_make_support::rustdoc;
 
 fn main() {
-    let output =
-        String::from_utf8(rustdoc().input("input.rs").arg("--test").command_output().stdout)
-            .unwrap();
+    let output = rustdoc().input("input.rs").arg("--test").run_fail();
+    let output = String::from_utf8(output.stdout).unwrap();
```

Comment:
This line will be all over the place. We could create our own `Output` wrapper with methods like `.stdout_utf8()` or something like that (doesn't need to be in this PR, ofc).

Reply:
Good point, added it to my list of TODOs :3